packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -42,24 +42,27 @@ defineOptions({ inheritAttrs: false })
 
 const floatingRef: Ref<HTMLElement | null> = ref(null)
 const wrapperRef: Ref<HTMLElement | null> = ref(null)
+const targetRef = ref<HTMLElement | undefined>(undefined)
 
-const targetRef = computed(() => {
```

Comment:
We should probably keep it as computed so it can be updated but we could do a check to see if the component is mounted 🤔

Reply:
good point, i updated accordingly!